packages-exp/auth-exp/src/core/strategies/anonymous.ts

```diff
@@ -30,7 +31,8 @@ export async function signInAnonymously(
     // If an anonymous user is already signed in, no need to sign them in again.
     return new UserCredentialImpl(
       auth.currentUser,
-      credential,
+      null,
```

Comment:
this signature is hard to read, should we pass a usercredentialparams in instead?

Reply:
alternatively maybe reorder the params so the optional ones go last?

Reply:
Done.